charts/trillian/README.md

```diff
@@ -1,46 +1,62 @@
-# Trillian
+# trillian
 
-[Trillian](https://github.com/google/trillian) is a log that stores an accurate, immutable and verifiable history of activity.
+<!-- This README.md is generated. Please edit README.md.gotmpl -->
```

Comment:
this we can use together with helm-docs, correct?

Reply:
correct.

See #300 on how this works.